docs/src/replay_attacks.md

```diff
@@ -0,0 +1,109 @@
+# Replay Attacks
+
+In general, a Replay Attack is
+
+> "a form of network attack in which valid data transmission is maliciously or fraudulently repeated or delayed." [[1]]
+
+Here we discuss replay attacks directly related to Mimblewimble transactions [[2]]. In this context, an adversary submits an exact duplicate of a previous transaction, with the goal of defrauding the receiver.
+
+In a blockchain based on the unspent transaction output (UTXO) model [[3]], a replay attack enables a _subsequent transaction_ to be performed for a second time without the permission of the UTXO holder.
+## Transaction Replay Attack Example
+
+In this example, Alice and Chuck collude to defraud Bob:
```

Comment:
```suggestion
In this example, Alice and Charlie collude to defraud Bob:
```
I know this is a massive nit, but I would prefer to have Chuck be called Charlie as Charlie is the widely used name for person C in crypto examples. 
Its always, Alice, Bob, Charlie, Dave, Eve

Reply:
Fair enough, but I chose Chuck for "A third participant, usually of malicious intent." [1], where Charlie is normally "A generic third participant."  

Not really too bothered so if you want me to change it I will!

1: https://en.wikipedia.org/wiki/Alice_and_Bob#Cast_of_characters